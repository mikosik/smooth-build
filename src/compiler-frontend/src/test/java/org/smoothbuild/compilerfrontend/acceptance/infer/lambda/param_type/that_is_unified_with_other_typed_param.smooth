myValue = (x, Int y) -> [x, y];
