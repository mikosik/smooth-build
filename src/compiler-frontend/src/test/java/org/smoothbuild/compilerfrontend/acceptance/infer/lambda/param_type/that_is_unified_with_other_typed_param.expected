(Int,Int)->[Int] myValue<>
  = (Int x, Int y) -> [x, y];
